src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/util/Extern.java

```diff
@@ -223,6 +226,134 @@ public boolean link(String url, String elemlisturl, Reporter reporter) throws Do
         return link(url, elemlisturl, reporter, true);
     }
 
+    /**
+     * Check whether links to platform documentation are configured. If not then configure
+     * links using the the documentation URL defined in {@code linkPlatformProperties} or the
+     * default documentation URL if that parameter is {@code null}.
+     *
+     * @param linkPlatformProperties path or URL to properties file containing
+     *                               platform documentation URLs, or null
+     * @param reporter The {@code DocErrorReporter} used to report errors.
```

Comment:
(pedantic) param descriptions are typically phrases (no initial capital, no trailing period) 
In this case, your two `@param` descriptions are inconsistent